text/0073-referedum-deposit-track.md

```diff
@@ -0,0 +1,70 @@
+# RFC-0073: Decision Deposit Referendum Track
+
+|                 |                                                                                |
+| --------------- |--------------------------------------------------------------------------------|
+| **Start Date**  | 12 February 2024                                                               |
+| **Description** | Add a referendum track which can place the decision deposit on any other track |
+| **Authors**     | JelliedOwl                                                                     |
+
+## Summary
+
+The current size of the decision deposit on some tracks is too high for many proposers. As a result, those needing to use it have to find someone else willing to put up the deposit for them - and a number of legitimate attempts to use the root track have timed out. This track would provide a more affordable (though slower) route for these holders to use the root track.
```

Comment:
>The current size of the decision deposit on some tracks is too high for many proposers.

I think in all cases these can be proposed on the whitelisted caller track instead and be fast-tracked by the fellowship.

Reply:
However, whitelisted caller has a minimum of 5% final support level compared to 0% on other tracks.  So you need at least a lot of abstentions (I think those count against the support threshold).

Reply:
I don't think everything that requires Root will go through Fellowship though, for example [457](https://polkadot.polkassembly.io/referenda/457). It requires Root but it's not a "pure, neutral" call.